examples/nlp/text_classification/text_classification_with_bert.py

```diff
@@ -12,203 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO: WIP
 
 import os
-from argparse import ArgumentParser
+
+import hydra
+from hydra.core.config_store import ConfigStore
```

Comment:
do we need this import? In general, please remove unused imports

Reply:
One of the first steps is to delete all arparse code!